lib/matplotlib/legend.py

```diff
@@ -4,29 +4,27 @@
 
 .. important::
 
-    It is unlikely that you would ever create a Legend instance
-    manually.  Most users would normally create a legend via the
-    :meth:`~matplotlib.axes.Axes.legend` function. For more details on legends
-    there is also a :doc:`legend guide </tutorials/intermediate/legend_guide>`.
-
-The Legend class can be considered as a container of legend handles and
-legend texts. Creation of corresponding legend handles from the plot elements
-in the axes or figures (e.g., lines, patches, etc.) are specified by the
-handler map, which defines the mapping between the plot elements and the
-legend handlers to be used (the default legend handlers are defined in the
-:mod:`~matplotlib.legend_handler` module).  Note that not all kinds of
-artist are supported by the legend yet by default but it is possible to
-extend the legend handler's capabilities to support arbitrary objects. See
-the :doc:`legend guide </tutorials/intermediate/legend_guide>` for more
-information.
+    It is unlikely that you would ever create a Legend instance manually.
+    Most users would normally create a legend via the `~.Axes.legend`
+    function. For more details on legends there is also a :doc:`legend guide
+    </tutorials/intermediate/legend_guide>`.
+
+The Legend class is a container of legend handles and legend texts. The legend
+handler map specifies how to create legend handle from artists (lines, patches,
+etc.) in the axes or figures; default legend handlers are defined in the
+:mod:`~matplotlib.legend_handler` module.
+
+The default legend handlers do not cover all artist types, but custom legend
+handlers can be defined to support arbitrary objects. See the :doc:`legend
+guide </tutorials/intermediate/legend_guide>` for more information.
```

Comment:
Different sectioning and minor formatting/wording/typo:
```suggestion
The `Legend` class is a container of legend handles and legend texts.

The legend handler map specifies how to create legend handles from artists
(lines, patches, etc.) in the axes or figures. Default legend handlers are defined
in the :mod:`~matplotlib.legend_handler` module. While not all artist types are
coverd by the default legend handlers, one can define custom legend handlers
to support arbitrary objects.

See the :doc:`legend guide </tutorials/intermediate/legend_guide>` for more
information.
```

Reply:
done (with small additional edits)